tools/sggcloud/tools.go

```diff
@@ -0,0 +1,33 @@
+package sggcloud
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"go.einride.tech/sage/sg"
+)
+
+// GHACredentials generates short-lived credentials for GCP with workload identity federation.
+func GHACredentials(ctx context.Context, projectNumber, serviceAccount string) string {
+	outputFile := filepath.Join(os.TempDir(), fmt.Sprintf("gha-creds-%s.json", projectNumber))
+	if _, err := os.Stat(outputFile); err == nil {
+		return outputFile
+	}
+	//nolint:lll
+	wifPool := fmt.Sprintf("projects/%s/locations/global/workloadIdentityPools/github-actions/providers/github-actions", projectNumber)
+	audienceURL := fmt.Sprintf("https://iam.googleapis.com/%s", wifPool)
+	if err := sg.Command(ctx, "gcloud", "iam", "workload-identity-pools", "create-cred-config",
+		wifPool,
+		"--service-account", serviceAccount,
+		"--output-file", outputFile,
```

Comment:
Any chance it is possible to supply a dash here (`-`) to get the output in the stdout rather than a file so we avoid having to go through disk writes in the host for this?

Reply:
This file is read by the GOOGLE_APPLICATION_CREDENTIALS, which only accept a file